tests/app.test.ts

```diff
@@ -1,8 +1,99 @@
 import request from "supertest";
-import  app from "../source/app";
+import router from '../routes/routes';
+import { Soldier } from "../repository/repo";
+const app = router; 
+beforeEach(async ()=>{ 
+    await Soldier.deleteMany();
+});
 
 describe('Is Soldier Alive', () => {
     it('Should Send an API Get Request If Soldier Is Alive, Expected 200:', () => {
         request(app).get('/health').expect("I am alive!");
     });
 });
+
+describe('Insert Soldier into DB #important: you need to modify the ID of inserted soldier in the test file in order to succeed this test', () => {
+    it('Should Send an API Post Request and save soldier to DB, Expected 201:',async () => {
+
+        const SoldierToBeAdded = {
+            "_id": "179",
+            "name": "cringus",
+            "rank":"Silver", 
+            "limitations": [
+              "Useless",
+              "no_one_likes_this_guy"
+            ]
+          }
+        await request(app).post('/soldiers').send(SoldierToBeAdded).expect(201);
```

Comment:
What else do you expect to happen when you insert a soldier? If I created a function that when you call insert with the correct schema it responds with 201, wouldn't that be enough?